scipy/stats/_stats_py.py

```diff
@@ -9059,8 +9112,14 @@ def wasserstein_distance(u_values, v_values, u_weights=None, v_weights=None):
     References
     ----------
     .. [1] "Wasserstein metric", https://en.wikipedia.org/wiki/Wasserstein_metric
-    .. [2] Ramdas, Garcia, Cuturi "On Wasserstein Two Sample Testing and Related
-           Families of Nonparametric Tests" (2015). :arXiv:`1509.02237`.
+    .. [2] https://lilianweng.github.io/posts/2017-08-20-gan/#what-is-
+           wasserstein-distance.
```

Comment:
Here's an example of references for [another function](https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.bootstrap.html):
![image](https://user-images.githubusercontent.com/6570539/210727080-ba1f4933-f49b-4abe-b21b-2d2cdeeb2077.png)

If you can add the author, I think this would be sufficient.

Reply:
This was marked as resolved, but it has not been resolved. The order is typically Author, name of article, name of publication, then URL.  So I would add the name of the author at the beginning of what you suggested.

Reply:
Okay now it looks like
```
    .. [2] Lili Weng, "What is Wasserstein distance?", Lil'log, 
           https://lilianweng.github.io/posts/2017-08-20-gan/#what-is-
           wasserstein-distance.
```